src/notifications/batterynotifier.cpp

```diff
@@ -153,6 +158,12 @@ void BatteryNotifier::prepareNotification()
             checkChargingTimer->start();
         }
     }
+    if (isStateChanged) {
+        if (!isAnyChargingState)
```

Comment:
In at least one case above, have used braces around one-line expression, so for consistency it may be best to use enclosing braces here too